debian/copyright

```diff
@@ -0,0 +1,37 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+
+Files: *
+Copyright: 2022 Nicolai Buchwitz <n.buchwitz@kunbus.com>
+License: GPL-2+
+ This package is free software; you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation; either version 2 of the License, or
+ (at your option) any later version.
+ .
+ This package is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+ .
+ You should have received a copy of the GNU General Public License
+ along with this program. If not, see <https://www.gnu.org/licenses/>
+ .
+ On Debian systems, the complete text of the GNU General
+ Public License version 2 can be found in "/usr/share/common-licenses/GPL-2".
+
+License: LGPL-3+
```

Comment:
What is now licensed with GPL-2 and what is LGPL-3+? Or is it dual-licensed?

Reply:
dual-licensed. The idea is to allow customers to integrate the device info data into their applications even if this closed.

Reply:
Why LGPL-3 not LGPL-2?
Also the License is missing in the source directory.

Reply:
@iluminat23 do you have a suggestion to add both licenses to the file? Append the second license? separate files with license name suffix?

Reply:
As fare a I know (L)GPL v3 requires, that an owner of the device must be enabled to run his changed code. What makes it the least favorite license on all embedded devices.
Hmmm I've never did anything like this. How does the rest of the world these things? If we know that we can ignore the rest of the world and do it in a sane way ;)
If we want to be permissive we could also use MIT. Then nobody has to care about any license incompatibilities.

Reply:
That would be a show-stopper. I'm not a professional licensee, so maybe MIT might be a better idea? I really don't care about anyone using this library for their profit maximization :)

Reply:
We could do this:
`debian/*` GPL-2
`*` MIT
This would be a real debian like way to do it.

Reply:
... an license file with MIT?